src/Bundle/Doctrine/ORM/Driver.php

```diff
@@ -15,10 +15,10 @@
 
 use Doctrine\ORM\EntityRepository;
 use Doctrine\Persistence\ManagerRegistry;
-use Doctrine\Persistence\ObjectManager;
 use Sylius\Component\Grid\Data\DataSourceInterface;
 use Sylius\Component\Grid\Data\DriverInterface;
 use Sylius\Component\Grid\Parameters;
+use Sylius\Resource\Exception\RuntimeException;
```

Comment:
Wrong exception atm.

```suggestion
use Sylius\Grid\Exception\RuntimeException;
```